### ISSUE TITLE
It Can NOT preprocess properly in SystemVerilog files?

### ISSUE THREAD

**Issue by zpxy** [chipsalliance/verible#2217]
**Describe the bug**
I use a gloabal _"define"_  file   and "macro" file to manager all parameters and macros, but verible-verilog_lint  can't recongnize the macro in other file , it say an error: **error in expanding the macor , might not be difined before**
![image](https://github.com/user-attachments/assets/1e5c0938-898a-4463-85cf-efb6249166eb)

so would you plan to solve the preprocess problem?

**Comment by hzeller** [comment]
Yes, preprocessor does need some work. It is fairly high on my list, but unfortunately there are other projects also competing for my attention.

**Comment by MinhTran0911** [comment]
I also face the same issue, especially when working with UVM since UVM relies heavily on macros. Is there a rough estimation on when this will be fixed?
